src/sinks/util/service.rs

```diff
@@ -64,18 +64,38 @@ impl<L> ServiceBuilderExt<L> for ServiceBuilder<L> {
     }
 }
 
-/// Tower Request based configuration
-#[derive(Clone, Copy, Debug, Deserialize, Serialize)]
+/// Reliability-related settings for outbound requests i.e. concurrency and rate limits, timeouts, etc.
```

Comment:
Would concurrency be considered a reliability setting? Maybe reliability isn't the right descriptor.

Reply:
Yeah, you're right... but I can't think of a better word off the top of my head. 🤔

Reply:
Maybe `Middleware settings for outbound requests`?

Reply:
Works for me.

Reply:
Addressed in 5cb42267b.